stdlib/public/SwiftRemoteMirror/SwiftRemoteMirror.cpp

```diff
@@ -635,3 +635,16 @@ swift_reflection_ptr_t swift_reflection_allocationMetadataPointer(
   NativeAllocation.Size = Allocation.Size;
   return Context->allocationMetadataPointer(NativeAllocation);
 }
+
+const char *swift_reflection_iterateMetadataAllocationBacktraces(
+  SwiftReflectionContextRef ContextRef,
+  swift_metadataAllocationIterator Call,
+  void *ContextPtr) {
+  auto Context = ContextRef->nativeContext;
+  auto Error = Context->iterateMetadataAllocationBacktraces([&]
+    (auto AllocationPtr, auto Count, auto Ptrs) {
+    std::vector<swift_reflection_ptr_t> ConvertedPtrs{&Ptrs[0], &Ptrs[Count]};
+    Call(AllocationPtr, Count, ConvertedPtrs.data(), ContextPtr);
```

Comment:
This now makes me wonder ... `ConvertedPtrs.data()` is effectively just `Ptrs`.  I think that this call is effectively:
```
Call(AllocationPtr, Count, Ptrs, ContextPtr);
```

Reply:
That's true only when `swift_reflection_ptr_t` and `StoredPointer` are the same. When they aren't (as is the case for 32-bit targets) then this is needed to convert the values.

Reply:
Ah, right, the foreign case.  A comment about that would be nice!

Reply:
You got it!